build-parent/pom.xml

```diff
@@ -39,7 +39,7 @@
         <!-- These properties are needed in order for them to be resolvable by the documentation -->
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
-        <graal-sdk.version-for-documentation>22.0.0.2</graal-sdk.version-for-documentation>
+        <graal-sdk.version-for-documentation>22.0</graal-sdk.version-for-documentation>
```

Comment:
Is that change necessary for documentation update?

Reply:
Reverted.